example/spec/parallel/testAPI/snsRuleSpec.js

```diff
@@ -116,6 +116,7 @@ describe('The SNS-type rule', () => {
         rule: snsRuleDefinition,
       });
       createdRule = JSON.parse(postRuleResponse.body);
+      expectedStatementId = getSnsTriggerPermissionId(createdRule.record);
```

Comment:
This PR (the one we're targeting here) is already updated to use `expectedStatementId`: https://github.com/nasa/cumulus/pull/3149

This change implements `getSnsTriggerPermissionId` to match the logic in main. Using ``${ruleName}Permission`` as in feature/rds-phase-3 does not match main and will cause issues for users once they take these changes.

Reply:
(happy to expand on those issues or talk through it)